services/worker/src/worker/job_runner.py

```diff
@@ -411,6 +411,7 @@ def compute(self) -> JobResult:
         pass
 
     # should be overridden if the job has children jobs of type "split"
```

Comment:
```suggestion

```
I think the comment is redundant here, since it is going to be an abstract method.

Reply:
Also, you could pass the result in line 425 instead of returning a set()